.buildbot.sh

```diff
@@ -2,19 +2,42 @@
 
 set -e
 
-make
+make bf_base
 
 export CARGO_HOME="`pwd`/.cargo"
 export RUSTUP_HOME="`pwd`/.rustup"
 
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
 sh rustup.sh --default-host x86_64-unknown-linux-gnu \
-    --default-toolchain stable \
+    --default-toolchain nightly \
     --no-modify-path \
     --profile minimal \
     -y
 export PATH=`pwd`/.cargo/bin/:$PATH
 
-rustup toolchain install stable
+rustup toolchain install nightly
```

Comment:
Is this needed? Above the default is already nightly?

Reply:
I don't know -- I'm not a rustup person!

Reply:
It looks to me like you can remove this step.

Reply:
Fixed in 1e56e63.